guides/wordpress-openid-connect-client-plugin-integration/index.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: doc
+title: "Wordpress Integration Guide"
+abstract: ""
+---
+
+This guide will step you through the process of adding DID to your Wordpress website using the ‘WordPress OpenID Connect Client' plugin by miniOrange.  If you have any difficulty completing the guide please email us for support: [team@did.app](mailto:team@did.app).
+
+You can also refer to a working example on our demo website: [wordpress.did.app](https://wordpress.did.app).  This guide assumes you have a working knowledge of Wordpress and a Wordpress website up and running.
+
+## Step 1: Create a new app in your DID account
+
+Visit [did.app](https://did.app), sign in and create a new app.  
+
+Give your new app a descriptive name e.g. ‘Wordpress Demo’ and use the full URL of the website in the ‘host’ box.
+
+Leave the ‘Test mode’ box unchecked.
+
+Click ‘Create App’.
+
+DID will now generate a Client ID and a Client Secret for your app.  Retain this information for step 3 (leave the tab open so you can come back to it).
+
+![screenshot of DID app configuration](/assets/images/wordpress-guide/configure-did.PNG)
+
+## Step 2: Install the plugin on your Wordpress website
+
+In your Wordpress admin dashboard, select __Plugins > Add New__ from the main menu.
+
+On the ‘Add Plugins’ page, search: ‘__WordPress OpenID Connect Client__’.  Look for the highlighted result in the screenshot below and click ‘Install Now’. Once the app has installed, click ‘Activate’.
+
+![screenshot of DID app configuration](/assets/images/wordpress-guide/plugin-library-screenshot.jpg)
+
+You can also visit the plugin’s page to download, install and activate manually if you prefer:
+
+[wordpress.org/plugins/miniorange-openid-connect-client/](https://wordpress.org/plugins/miniorange-openid-connect-client/)
+
+## Step 3. Configure the plugin
+
+Once activated, the plugin creates a menu option called ‘miniOrange OpenID Connect’.  Click on this to start configuring the plugin to work with DID.
+
+![screenshot of miniOrange OpenId connect plugin menu option](/assets/images/wordpress-guide/plugin-in-menu.jpg)
+
+You will be presented with a screen which looks like this:
+
+![screenshot of miniOrange OpenId connect plugin config dashboard](/assets/images/wordpress-guide/miniOrange-config-panel.jpg)
+
+Scroll right to the bottom on this page and look for the option called ‘__Custom OpenID Connect App__’ under the heading ‘Custom Applications’.  It looks like this:
+
+![screenshot of miniOrange OpenId connect plugin config dashboard](/assets/images/wordpress-guide/custom-applications.PNG)
+
+Select ‘Custom OpenID Connect App’.  You will now see this screen:
+
+![screenshot of miniOrange OpenId connect plugin config dashboard](/assets/images/wordpress-guide/plugin-config.jpg)
+
+Complete the fields using the following values:
+
+```
+App Name: DID
+Client ID: *copy and paste this from your DID app settings page.*
```

Comment:
It's a shame the styling doesn't look quite right here.
Could it just show these values written into the correct boxes in the screenshot?

Reply:
the other option is to have it as a list
```
- **App Name:** DID
```

Reply:
list